_posts/2019-09-23-good-first-issue.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "How I built Good First Issue bot with OpenFaaS Cloud"
+description: In this post Rajat will share what triggered him to write goodfirstissue bot, where was it deployed and how it was an easy decision to move it to OpenFaaS cloud.
+date: 2019-09-23
+image: /images/good-first-issue/yellow-stone.jpg
+categories:
+  - bot
+  - golang
+  - go
+  - twitter
+author_staff_member: rajatjindal
```

Comment:
This should be `alex` right? 

Or maybe that will get updated before the merge.

The way it is, it sounds strange at the end "follow me @alexellisuk" while there is a picture of Rajat.

Reply:
thats right. thank you for catching it. Fixed.